hpke/hybrid-x25119-kyber768-test-vectors.json

```diff
@@ -0,0 +1 @@
d564792cee3d0bf4594f67348995545020ae184b8680c6c020d831e4b5279ced8439538e6c74613d656f17240f82108d801b95b37efcd5c2c6e9e75cb54153cd5b07a56dc01d38c38d37558565e5248a616993c8adccf613b96e5d597aecb877797907815d654c95df6acef0fe37e074b3f9019c7f259e8349c25ec792b4f3c85c1de5c45787ad07d08e3de55394db122843555fce311aad6581a1b88d75932cc3ad6e1b41e2e03aadcf989b50f5505e840f6d019b705f517c48b5c42b1c4bdc61ec2d2d710302fb763cc93cea27933eb2102ae6f7d664f354bebef0c5bd9ccdf366e5c89c6248b80994c3690ca19735f959220581749046fd54ed0931a82d99cbd8d6c9155fa53c73315a1a74ea7ce95ca4b6c98cb68152ef9b833ca183a4c0a087b01cd863c1b3191910da1022ef45d43d54358ab365067dd1dabcb17751042ec5f276414caa0e71296c212a69b2d19df0bbf51e34d57a00fb43efa9c86d9eb29bb8bf6f4ccd2343f2da9e4488957e40681770e2f86ee12390d11f5ae038625f65a5ed2acfd02aef786c38b5d9f5966d0975512deffe4dbe136616e901320b87ec30719928e19c43c601eb1b47d1d08c55a49a534a10b9258298494ad1e8631a1ebc2a41226e1fcd9ba6fffd531a366c323a95ae94183f9504a09b7828c59681085e556e3f0d8a94b2a3b9a4811b973e466b16c14cb3710cbc15e1810ed3679e53c93f7c1c262abb84a9170dbca084081e806caf1660260ce0aed84974252e519bc8e718ebdff059cc7819af37b9a89c3afbe4d8767dce4e199bd2e56370199850ed551f97823219a58cdf74faa933aec85768005d95d2afbdc5916d9935bd8b56b3d64fa48b3d39e047e40591b4b60d4a1b33074c6ee563b14e2ea39c809d51a026f8575cd8da6fd814d9a3cb834b252a4a7d9ce6f50c182c3cf9e780e348bffc3884e0727375be44f6f08af1adbe6ebd8ef4e62f04999bc7246ff1bd621e9e184e66e6c4273a703c3a61aad1d6809a3020cbb57d329a5a9e0397468a167a2191e660ae95bdcc370703882316e7f1dd896d3a3466a7ccac5c88e129aa173432409b1bd2b8abcbd034430a7983c5b31cc12114bc8a0fc94aad2d6991389aaf9e74adc3abf604503dd499b61849c9936c3feacbe4f0ee5852119c65ca2f23343dcf2579c689b0cba265073de2546913dde87a2c3d55c7a5cee88c78be125defb4436ca1576fc7c163cd06f01fc00de5cb32981842a086e2b75e0125c4f87e6229132eb18cde901f296f0c050e9586bd5a7f12f8f087cab9d2164ac41102062619a314639c051787e5dff24896c7e9ed7fe299a598fabc9dd15671e26e744a514ca6b0328235c919def4ad8a835e87f41519a661d1fc8f59c85f3c9ca8544325ca0932abc6bea33f47e43a84e1d","shared_secret":"6cf976165667e7da5d06ddbb7840f8d766609ee1a3f3bd1c947a4498afb4c4ac9137e2e99a3bb83da5c01108f30150e33c804300c3032d212a0f63c2d6bdd631","key_schedule_context":"009f749a195d1c8b3eaa8d5c3f571dc7231aafbbc0405e4b484738352667c4848677d1a0316e6519bc5ce5bb9ab1eaad121d47115eeecde2c60abef07a70ada444","secret":"eb07f25547b772c34e6258b21be0c29bd8d342fc1c1971a71ee8adea30ddd678","key":"8bfee5330f68713fc27fb74d4876f97c","base_nonce":"99c08a3cda9f6feb763dcec8","exporter_secret":"850da245cff2a8bc02f1dc2f6ea66bea376c92eba09a47a89e392abfe6a91619","encryptions":[{"aad":"6164646974696f6e616c207075626c69632064617461","ct":"c2c92b8c097ed475b0e26fa4e409ce570965c4eb6d7804ecfdfe423f1dc9c9949cc15b8d50e1aa9a1e6fddebe814b2ea8834","nonce":"99c08a3cda9f6feb763dcec8","pt":"7465787420656e6372797074656420746f20426f622773207075626c6963206b6579"}],"exports":[{"exporter_context":"7465737420636f6e74657874","L":32,"exported_value":"4824948ac0b9d9a47c4ee80d3d0269006bdbe18f637df9d760fa77b1ca19a023"}]}]
```

Comment:
Did you intend to commit this file?

Reply:
Yeah, but we can pull it out if we don't want it in the repo.